libs/api/database/src/lib/content/stores/browse.store.ts

```diff
@@ -74,10 +71,10 @@ export class BrowseStore {
         pageNum: number,
         kinds: ContentKind[],
         filter: ContentFilter,
-        userId?: string
+        userId?: string,
     ): Promise<PaginateResult<ContentDocument>> {
         const query = { kind: { $in: kinds }, 'audit.isDeleted': false, 'audit.published': PubStatus.Published };
-        const filteredQuery = await this.determineContentFilter(query, filter);
+        const filteredQuery = await BrowseStore.determineContentFilter(query, filter);
```

Comment:
Why replace "this" with "BrowseStore" throughout this file?

Reply:
recommendation of my IDE, to be honest